macros/cross_db_utils/current_timestamp.sql

```diff
@@ -3,7 +3,7 @@
 {% endmacro %}
 
 {% macro default__current_timestamp() %}
-    current_timestamp::{{dbt_utils.type_timestamp()}}
+    getdate()::{{dbt_utils.type_timestamp()}}
```

Comment:
I think we'll want to preserve `current_timestamp` in the default implementation, but you can totally provide a redshift-specific implementation similar to the bigquery one below. That could just look like:
```
{% macro redshift__current_timestamp() %}
  getdate()::{{dbt_utils.type_timestamp()}}
{% endmacro %}
```

The `default` implementation here is going to be used by other databases like Snowflake and Postgres which do not (AFAIK) support the `getdate()` function.